index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Stuart Dambi | Fullstack Developer</title>
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/1a66ebfe2a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header class="header">
+    <ul>
+      <li><a href="#" class="developer">Dambi Stuart</a></li>
+      <li><a href="#"><i class="fa-solid fa-bars 2x"></i></a></li>
+    </ul>
+  </header>
+
+  <section class="welcome-area">
+    <h1 class="title">Hey there. <br> I'm Stuart</h1>
+    <h3 class="subtitle">I'am a Software Developer</h3>
+    <p class="description">
+      I can help build a product, feature or website.
+      Look through some of my work and experience! If you like what you see
+      and have a project you need coded, don't hesitate to contact me.
+    </p>
```

Comment:
Good job, but make sure to follow the font size of the Figma design. You can resize your text size for this purpose.


![image](https://user-images.githubusercontent.com/34779928/191606119-d7726bb9-be8e-47e4-a6b4-63c87d6c94f4.png)

Reply:
@codedbystuart Kindly check this comment 🤗